src/index.ts

```diff
@@ -16,11 +16,13 @@
 // ** https://github.com/googleapis/gapic-generator-typescript **
 // ** All changes to this file may be overwritten. **
 
+import * as v1 from './v1';
 import * as v1beta1 from './v1beta1';
+const BigQueryReadClient = v1.BigQueryReadClient;
```

Comment:
So I think that's a breaking change :) Unless the v1 endpoint is identical to v1beta1?

Reply:
@JustinBeckwith No, they're not identical but Seth was saying that I don't need to handle it as a breaking change since the version is included in the import and it's not expected that users will automatically transition to v1. Do you have other thoughts about that?

Reply:
To me it actually looks like a new feature, not a breaking change. It only had `BigQueryStorageClient`, but now it also has `BigQueryReadClient`.